pkgs/build-support/setup-hooks/make-binary-wrapper/default.nix

```
@@ -19,7 +19,7 @@ makeSetupHook {
   passthru = {
     # Extract the function call used to create a binary wrapper from its embedded docstring
     extractCmd = writeShellScript "extract-binary-wrapper-cmd" ''
-      strings -dw "$1" | sed -n '/^makeCWrapper/,/^$/ p'
+      ${cc.targetPrefix}strings -dw "$1" | sed -n '/^makeCWrapper/,/^$/ p'
```

```suggestion
      ${cc.bintools.targetPrefix}strings -dw "$1" | sed -n '/^makeCWrapper/,/^$/ p'
```
strings is from bintools